pkg/management/postgres/instance.go

```diff
@@ -247,11 +251,41 @@ func (instance *Instance) SetCanCheckReadiness(enabled bool) {
 	instance.canCheckReadiness.Store(enabled)
 }
 
+// HasWALDiskSpace checks if we have enough disk space to store two WAL files
+func (instance *Instance) HasWALDiskSpace(ctx context.Context) (bool, error) {
```

Comment:
How do we know that the issue is the "WAL disk"? If the cluster does not have a separate volume for WAL storage, then this would be the PG_DATA disk, no?

Reply:
Yes! But that was meant to check if there's free space for WALs, besides in which volume they're hosted.

Reply:
I know that what we mean here is "Space on the disk to store WALs", but the workding is "Has space on the WAL disk". It's close but not the same, and my reflex impression before reading the code is that this requires the dedicated WAL volume.
How about `HasDiskSpaceForWAL`?
Or, if we keep the method name, we can add a comment to clarify